src/app/core/models/leasing.model.ts

```diff
@@ -0,0 +1,15 @@
+import { User } from './user.model';
+import { Product } from './product.model';
+import { LeasingStatusFromServer } from 'src/app/shared/components/order-status/order-status.component';
+
+export type Leasing = {
+  leasingID: string;
+  transactionID: string;
+  lessor: User;
+  lessee: User;
+  product: Product;
+  status: LeasingStatusFromServer;
+  startDate: Date;
+  endDate: Date;
+  price: number;
```

Comment:
Redundant. The price is located under the product

Reply:
Done